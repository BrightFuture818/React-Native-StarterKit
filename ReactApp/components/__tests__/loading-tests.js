/**
 * Test to check if the component renderes correctly
 */
'use strict';

import 'react-native'
import React from 'react'
import Loading from '../loading'
import renderer from 'react-test-renderer'

/**
 * Check if alerts renders correcly
 * and asserting it to the matching snapshot
 */
it('Loading renders correcly', () => {

  const tree = renderer.create(
    <Loading />
  ).toJSON()

  expect(tree).toMatchSnapshot()

});
