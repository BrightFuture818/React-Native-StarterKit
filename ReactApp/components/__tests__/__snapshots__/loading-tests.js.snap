exports[`test Loading renders correcly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#FFFFFF",
        "flex": 1,
        "flexDirection": "column",
        "position": "relative",
      },
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      },
      undefined,
    ]
  }>
  <ActivityIndicator
    animating={true}
    color="#AAA"
    hidesWhenStopped={true}
    size="large" />
  <View
    style={
      Array [
        Object {
          "height": 1,
          "left": 0,
          "marginTop": 10,
          "right": 0,
        },
      ]
    } />
</View>
`;
